tests/test_updater_top_level_update.py

```diff
@@ -339,6 +339,35 @@ def test_new_timestamp_expired(self) -> None:
 
         self._assert_files_exist([Root.type])
 
+    def test_new_timestamp_fast_foward_recovery(self) -> None:
+        self._init_updater()
+
+        # health initial version of the timestamp
+        self._run_refresh()
+        self._assert_version_equals(Timestamp.type, 1)
+
+        # attacker updates to a higher version and add new package
```

Comment:
I don't understand "add new package"?

Reply:
I added all the scenarios/steps, such as adding packages before/after. I simplified it, and  I left it by mistake.